app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -77,191 +112,531 @@ fun RouteScreen(
                 true
             }
         )
+
     val coroutineScope = rememberCoroutineScope()
 
+
+    /**
+     * Separate bottom sheets for start query and destination query, wrapping composable of
+     * entire main menu screen
+     */
     ModalBottomSheetLayout(
-        sheetState = sheetState,
+        sheetState = startSheetState,
         sheetContent = {
             RouteOptionsSearchSheet(routeViewModel, onCancelClicked = {
                 coroutineScope.launch {
-                    sheetState.hide()
+                    startSheetState.hide()
                 }
-            })
+            }, onItemClicked = {
+                coroutineScope.launch {
+                    startSheetState.hide()
+                }
+            }, isStart = true)
         },
         sheetShape = RoundedCornerShape(8.dp)
     ) {
-        Column(modifier = Modifier.fillMaxSize()) {
-            //TODO make an AppBarColors class w/ the right colors and correct icon
-            TopAppBar(
-                title = {
-                    Text(
-                        text = "Route Options",
-                        fontFamily = sfProDisplayFamily,
-                        fontStyle = FontStyle.Normal
-                    )
-                },
-                navigationIcon = {
-                    IconButton(onClick = { navController.popBackStack() }) {
-                        Icon(
-                            imageVector = Icons.AutoMirrored.Outlined.KeyboardArrowLeft,
-                            contentDescription = ""
-                        )
+        ModalBottomSheetLayout(
+            sheetState = destSheetState,
+            sheetContent = {
+                RouteOptionsSearchSheet(routeViewModel, onCancelClicked = {
+                    coroutineScope.launch {
+                        destSheetState.hide()
                     }
+                }, onItemClicked = {
+                    coroutineScope.launch {
+                        destSheetState.hide()
+                    }
+                }, isStart = false)
+            },
+            sheetShape = RoundedCornerShape(8.dp)
+        ) {
```

Comment:
Woah, weird. A `ModalBottomSheetLayout` IN a `ModalBottomSheetLayout`? Wacky. I guess if this is most convenient then it's ok.